config/src/vespa/config/frt/frtconnectionpool.cpp

```diff
@@ -97,7 +97,7 @@ namespace {
  * @return the hash value
  */
```

Comment:
This is a very bad hash function. Since hostnames are often very similar I strongly suggest using xxhash.
It says here that the hash function should be the same in both java/cpp, but I do not see why.

Reply:
We will address this in a later PR.